src/models/queues.ts

```diff
@@ -73,6 +74,15 @@ export class Queue {
      */
     @prop()
         text_channel?: string;
+
+    /**
+     * Text Channels ids to log queue events
+     */
+    @prop({ default: [], required: false })
+        info_channels!: {
+            channel_id: string;
+            events: QueueEvent[];
```

Comment:
QueueEvent should be a db model if you want to use it here. Also i think it makes more sense here to log the info channel inside the event, so we don't have to nest here. That would also justify it not being only an Enum.